subsys/spm/spm.c

```diff
@@ -382,10 +384,9 @@ static void spm_configure_ns(const tz_nonsecure_setup_conf_t
 	/* Allow SPU to have precedence over (non-existing) ARMv8-M SAU. */
 	tz_sau_configure(0, 1);
 
-#if defined(CONFIG_ARMV7_M_ARMV8_M_FP)
+#if defined(CONFIG_ARMV7_M_ARMV8_M_FP) && CONFIG_SPM_NRF_FPU_NS
```

Comment:
If these are both `bool` kconfig options, it'd be more consistent to check them similarly, e.g. either use `#if defined` for both or use just `#if` for both.

Reply:
Fixed